sdk/storage/azure-storage-blob-stress/src/main/java/com/azure/storage/blob/stress/utils/OriginalContent.java

```diff
@@ -70,10 +74,26 @@ public Mono<Boolean> checkMatch(Path downloadPath, Context span) {
             });
    }
 
-    private static Mono<Long> calculateCrc(Path downloadPath) {
-        return BinaryData
-            .fromFile(downloadPath)
-            .toFluxByteBuffer()
+   public Mono<Boolean> checkMatch(Flux<ByteBuffer> data, Context span) {
```

Comment:
nit: maybe `BinaryData` so it's easier to use?
```suggestion
   public Mono<Boolean> checkMatch(BinaryData data, Context span) {
```